src/tokens/TokensService.ts

```diff
@@ -54,27 +55,17 @@ export class TokensService {
   }
 
   getTokenByName(token: string): Token {
-    const knownToken = this.tokenRegistry.find((t) => t.name.toLowerCase() === token.toLowerCase());
-    if (!knownToken) throw new InternalServerError(`${token} definition not in TokenRegistry`);
+    const knownToken = ethTokens.find((t) => t.name.toLowerCase() === token.toLowerCase());
```

Comment:
I'd just make a `ethTokens.nameLower`

Also, no need to token.toLowerCase() in a loop...